packages/google_sign_in/google_sign_in/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 ## 5.2.2
 
 * Updates Android compileSdkVersion to 31.
-* Removes depenedncy on `meta`.
+* Bump Android dependency on com.google.android.gms:play-services-auth and remove the need for jetifier.
```

Comment:
Looks like you merged with another 5.2.2 bump; you'll need to move this to 5.2.3.

Reply:
done.